src/locale-strings.js

```diff
@@ -9,12 +9,15 @@ const localeStringData = {
     "en-GB": `A Primer on Bézier Curves`,
     "ja-JP": `ベジェ曲線入門`,
     "zh-CN": `贝塞尔曲线底漆`,
+    "uk-UA": `Підручник з кривих Безьє`,
   },
   subtitle: {
     "en-GB": `A free, online book for when you really need to know how to do Bézier things.`,
+    "uk-UA": `Безкоштовна онлайн-книга, яка навчить вас всьому необхідному, щоб працювати з кривими Безьє.`,
   },
   description: {
     "en-GB": `A detailed explanation of Bézier curves, and how to do the many things that we commonly want to do with them.`,
+    "uk-UA": `Детальне пояснення кривих Безьє та можливостей їх застосування.`,
   },
   longDescription: {
     "en-GB": `
```

Comment:
if possible, please also localize this field, and any other value with an `en-GB` entry